docs/4.1. Behaviour - Clients.md

```diff
@@ -19,33 +19,93 @@ Clients SHOULD support a mechanism for providing the information required for OA
 containing the required details).
 
 Clients are RECOMMENDED to provide for the OAuth 2.0 Dynamic Client Registration Protocol [RFC 7591][RFC-7591] to allow
-for zero-configuration setup.
+for zero-configuration setup. NMOS Nodes SHOULD be capable of registering dynamically in order to limit the
+configuration associated with large numbers of Nodes coming online simultaneously.
+
+Clients that do not register with an explicit `token_endpoint_auth_method` parameter will be registered as a
+Confidential Client and issued with a client secret. Public clients MUST specify a value of "none" for this parameter
```

Comment:
Is "none" defined in an RFC for this parameter?

Reply:
```
RFC 7591: "none": The client is a public client as defined in OAuth 2.0,
         Section 2.1, and does not have a client secret."
```